### ISSUE TITLE
the functional information for most genera is missing

### ISSUE THREAD

**Issue by martanierychlo** [cmc-aau/midasdashboard#9]
The functional information displayed in Shiny is missing all the details from the online version and contains only several green (POS) dots ; please change that back so that the information is pulled directly from midasfieldguide; otherwise we are missing majority of information

**Comment by KasperSkytte** [comment]
This was likely introduced in 2d6f5208b6dd8df23566a6c4b3e3a3c8c06d13db with the file you provided. Can you check that first please

**Comment by martanierychlo** [comment]
correct, the file only provided the information that in my understamnding was used to overwrite the info from midasfieldguide, while keeping the others unchanged. However I can see that all the remaining info from midasfieldguide are missing. In this case I would like to request to go back to previous solution (only info from midasfieldguide, without my custom file)

**Comment by KasperSkytte** [comment]
Sure. Would be best to have everything there up to date anyways
